### PR TITLE
Calckey support

### PR DESCRIPTION
**Pull request by ThatOneCalculator**
Merge https://codeberg.org/calckey/megalodon

### CODE REVIEW

File: example/typescript/src/misskey/timeline.ts [hunk 1]
```diff
@@ -1,20 +1,20 @@
...
   env: {
     MISSKEY_ACCESS_TOKEN: string
   }
 }
 
-const BASE_URL: string = 'https://misskey.io'
+const BASE_URL: string = 'https://chkukfsatpl3.cutls.dev'
 
-const access_token: string = process.env.MISSKEY_ACCESS_TOKEN
+const access_token: string = process.env.MISSKEY_ACCESS_TOKEN || '8zrWrxUIBNQ0kWUw'
```
Comment (h3poteto) [inline example/typescript/src/misskey/timeline.ts:11]: Is this a raw access token? Is it no problem?

**Comment by h3poteto** [comment]
Sorry, I can't merge this PR because too big and it includes changes other than Calckey. Can I create a new PR  dedicated to Calckey based on this PR?

File: example/typescript/src/misskey/timeline.ts (hunk 1 above)
Comment (ThatOneCalculator) [reply example/typescript/src/misskey/timeline.ts:11]: It's an example token

**Comment by ThatOneCalculator** [comment]
The Misskey fixes in this PR are also needed for Calckey, but you can create a new PR from this.

**Comment by Y0ngg4n** [comment]
@ThatOneCalculator Any news about this PR?

**Comment by ThatOneCalculator** [comment]
Don't ask me, ask the maintainer. For now, I maintain my own fork.

**Comment by Y0ngg4n** [comment]
@h3poteto any news?

**Comment by h3poteto** [comment]
There is nothing.
I don't have much motivation for this feature right now.

**Comment by ThatOneCalculator** [comment]
@Y0ngg4n if you want a version of megalodon that supports Firefish (formerly Calckey), here's our version of megalodon in our monorepo: https://git.joinfirefish.org/firefish/firefish/-/tree/develop/packages/megalodon?ref_type=heads

**Comment by h3poteto** [comment]
Thank you. Firefish support has started.

**Pull request closed without merging**